README.md

```diff
@@ -1,4 +1,48 @@
-CraftedWeb
-==========
+# CraftedWeb
 
-A CMS for game servers 
\ No newline at end of file
+CraftedWeb is a CMS for World of Warcraft Servers.
+
+## Features
+
+CraftedWeb has a plethora of features, outlined below.
+
+### Expansions
+|Expansion			| Supported?                    | Planned?                   |
+|------------------|-------------------------------|----------------------------|
+|Vanilla           |`No`            				| `Yes`          				|
+|Burning Crusade   |`No`            				| `Yes`         					|
+|WoTLK             |`Yes`							|							|
+|Cataclysm         |`No`							|`Yes`							|
+|Mists of Pandaria |`No`							|	`Yes`						|
+|WoD               |`No`							|`No`						|
+|Legion            |`No`							|		`Yes`					|
+
+### Main Features
+| Feature				| Supported	| Planned	|
+|-----------------------|-----------|-----------|
+| Account Management	|`Yes` 		|			|
+| Votes & Donations		|`Yes`		|			|
+| Character Tools		|`Yes`		|			|
+| Tooltip System		|`Yes`		|			|
+| Realm Status			|`Yes`		|			|
+| Multi-Realm Support	|`Yes`		|			|
+| Admin Panel			|`No*`		|`Yes`		|
+| Staff Panel			|`No*`		|`Yes`		|
+| Notification System	|`Yes`		|			|
+| News System w/ Comments|`Yes`		|			|
+| Template System		|`Yes`		|			|
+| Plugin Support		|`Yes`		|			|
+| WoWHead Integration	|`Yes`		|			|
+| OpenWoW Integration	|`Yes`		|			|
+| phpBB Integration		|`Yes**`	|			|
+
+> \* Feature exists, is not up-to-date
```

Comment:
After the comma e.g. "," add the following "is not fully functional at the moment".